src/commands/math.ts

```diff
@@ -318,6 +318,12 @@ class MathCommand extends MathElement {
   }
   mathspeakTemplate = [''];
   mathspeak() {
+    // this one runs before a full latex symbol has been formed;
+    //   check if there is more than backslashes in the latex
+    const tex = this.latex().trim();
+    if (this.mathspeakOverride && !/^(\\)+$/.test(tex)) {
```

Comment:
`/^(\\)+$/` will not match the empty string, so `mathspeakOverride` will be called if `tex` is empty. Is that okay? The comment  ("more than backslashes") isn't _quite_ accurate because of this edge case.